src/AnnotationProcessor/DoctrineAnnotationProcessor.php

```diff
@@ -205,27 +212,41 @@ private function transformType($type)
     {
         if ($type === Type::BOOLEAN) {
             return 'boolean';
-        } elseif ($type === Type::SMALLINT || $type === Type::BIGINT || $type === Type::INTEGER) {
+        }
+
+        if ($type === Type::SMALLINT || $type === Type::BIGINT || $type === Type::INTEGER) {
             return 'integer';
-        } elseif ($type === Type::FLOAT) {
+        }
+
+        if ($type === Type::FLOAT) {
             return 'float';
-        } elseif ($type === Type::TEXT || $type === Type::GUID || $type === Type::STRING || $type === Type::DECIMAL) {
+        }
+
+        if ($type === Type::TEXT || $type === Type::GUID || $type === Type::STRING || $type === Type::DECIMAL) {
             return 'string';
-        } elseif ($type === Type::BLOB /* binary will be added in doctrine 2.5 */) {
+        }
+
+        if ($type === Type::BLOB /* binary will be added in doctrine 2.5 */) {
             return 'resource';
-        } elseif (in_array(
+        }
+
+        if (in_array(
```

Comment:
why is this not transformed into `\in_array()`?

Reply:
Since there is no sniff for that. I didn't add it since it marked every global function as `\` not just the once that profited from the change.

Reply:
Then why is it added here? https://github.com/hostnet/accessor-generator-plugin-lib/pull/30/files#diff-013df530544b9b55b12bfa6b02531e8bR150

Reply:
Because I manually edited that file to get the tests working again and changed that.